pkg/datastore/mysql/iterator.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql
+
+import (
+	"database/sql"
+
+	"github.com/pipe-cd/pipe/pkg/datastore"
+)
+
+// Iterator for MySQL result set
+type Iterator struct {
+	cur *sql.Rows
```

Comment:
nit: IMO, `rows` is more readable than `cur`

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/1678/commits/31d25ab8c9f4ab0739e7468c947fb7141c33079e 🙏